src/config/2020.json

```diff
@@ -2,7 +2,7 @@
   "settings": [
     {
       "is_live": true,
-      "supported_languages": ["en","es","fr","hi","ja","nl","pt","ru","zh-CN","zh-TW"],
+      "supported_languages": ["en","es","fr","it","hi","ja","nl","pt","ru","zh-CN","zh-TW"],
```

Comment:
Good stuff! Was about to ask! Can you also add to `server/language.py` in this object:

```py
# Mostly we use region-agnostic languages, but can add region where
# there are significant differences (e.g. Chinese)
class Language(object):
    EN = _Language('English', 'en')
    ES = _Language('Español', 'es')
    FR = _Language('Français', 'fr')
    HI = _Language('हिन्दी', 'hi')
    JA = _Language('日本語', 'ja')
    NL = _Language('Nederlands', 'nl')
    PT = _Language('Português', 'pt')
    RU = _Language('Русский', 'ru')
    ZH_CN = _Language('简体中文', 'zh-CN')
    ZH_TW = _Language('繁體中文', 'zh-TW')
```